CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+### Added
```

Comment:
Also, a very good changelog you can use as a reference is this one:
https://github.com/python-poetry/poetry/blob/master/CHANGELOG.md?plain=1

E.G. something about documentation (like adding a changelog) should be under its own `Docs` section

Reply:
@John15321 Formatting should now be valid

Reply:
@John15321 Resolved issues raised